bin/autoaggregate

```diff
@@ -33,47 +34,61 @@ def aggregate(config):
         Path where to find the ``repos.yaml`` file.
     """
     logger.info("Running gitaggregate with %s", config)
-    old_umask = None
-    try:
-        # Download git code with the specified umask, if any
-        if UMASK:
-            old_umask = os.umask(int(UMASK))
-        check_call(
-            [
-                "gitaggregate",
-                "--expand-env",
-                "--config",
-                config,
-                "--log-level",
-                log_level,
-                "--jobs",
-                str(cpu_count() or 1),
-                "aggregate",
-            ],
-            cwd=SRC_DIR,
-            stderr=sys.stderr,
-            stdout=sys.stdout,
-        )
-    finally:
-        # Restore umask, if changed
-        if old_umask is not None:
-            os.umask(old_umask)
-        # Chown recursively, if UID or GID are specified
-        if ~UID or ~GID:
-            for root, dirs, files in os.walk(SRC_DIR):
-                for target in dirs + files:
-                    try:
-                        target_path = os.path.join(root, target)
-                        if not os.path.islink(target_path):
-                            # if target_path is a link we would only chown the
-                            # target, as the target should be chowned separately
-                            # we can skip all symlinks and prevent showing
-                            # unclear errors
-                            os.chown(target_path, UID, GID)
-                    except Exception:
-                        logger.debug(
-                            "Error trying to chown on file. Skipping...", exc_info=True
-                        )
+    pre_execs = []
+
+    def pre_exec():
+        for _exec in pre_execs:
+            try:
+                _exec()
+            except Exception as e:
+                logger.error("Error in %s: %s" % (_exec, e))
+                logger.exception(e)
+                raise
+
+    if UMASK:
```

Comment:
`UMASK` can not be evaluated `False` but cloud be falsy. E.g. you can launch autoaggregate with `UMASK="" autoaggregate` to prevent setting a umask for checkout.